data/Sunkio.json

```diff
@@ -0,0 +1,24 @@
+{
+  "name": "Tanja Schmidt",
+  "displayStatsPublic": true,
+  "type": "personal",
+  "bio": "Web & decentralized tech enthusiast. Software Developer keen on building web & blockchain solutions.",
+  "avatar": "https://github.com/Sunkio.png",
```

Comment:
```suggestion
```

Reply:
Thanks for your time and your feedback! I removed the link. Hope everything's okay now. :)